tools/readaheadstat_example.txt

```diff
@@ -0,0 +1,32 @@
+Demonstration of readaheadstat, the Linux eBPF/bcc version
+
+Read-ahead mechanism is used by operation sytems to optimize sequential operations
+by reading ahead some pages to avoid more epensive filesystem operations. This tool
+shows the performance of the read-ahead caching on the system under a given load to 
+investigate any caching issues. It collects stats for unused pages in the cache and
+also prints a histogram showing how long they have remianed there. 
```

Comment:
I guess we need more explanation about *when* people should use this tool. What kind of performance issues may trigger people to use this tool to check readahead performance.

Reply:
Explained this with a usage scenario as well